core/templates/domain/exploration/StatesObjectFactory.ts

```diff
@@ -152,10 +152,7 @@ export class States {
 })
 export class StatesObjectFactory {
   constructor(private stateObject: StateObjectFactory) {}
-  // TODO(ankita240796): Remove the bracket notation once Angular2 gets in.
-  /* eslint-disable dot-notation */
   createFromBackendDict(statesBackendDict: any): States {
```

Comment:
Why is this till any?

Reply:
Yeah, missed this file. Will do it right away.

Reply:
Will declare a type for it in my further prs. Thanks!